ament_lint_cmake/package.xml

```diff
@@ -11,6 +11,7 @@
   <license>Apache License 2.0</license>
 
   <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
```

Comment:
Are there any cases which are covered by pep8 / pep257 / pyflakes which are not covered by flake8? If not I would suggest to remove the redundant tests (ament_pep8, ament_pep257, ament_pyflakes).